### ISSUE TITLE
"i" in italic has an inconsistent baseline

### ISSUE THREAD

**Issue by demoy** [huertatipografica/Alegreya-Sans#9]
![screenshot_2017-10-14_00-13-10](https://user-images.githubusercontent.com/2396524/31572876-4cbb9df0-b075-11e7-93fe-906337ee6ffe.png)

"i" appears like a descender (at large font-sizes), but when paired in the "fi" ligature, it isn't .
I'm using Ubuntu 16.04, if that matters. Could you also tell me which of these two appearance is intentional? so I know what to expect a "fix" to look like..

**Comment by juandelperal** [comment]
Thanks @demoy
I can't reproduce your issue. Are you using the last version from this repo?

[image: Imágenes integradas 1]


2017-10-14 2:26 GMT-03:00 demoy <notifications@github.com>:

**Comment by demoy** [comment]
Using Alegreya-Sans-2.002. https://netrunnerdb.com/en/card/12016, was the linked screenshot, firefox was used. I used firefox to force alegreya sans to replace the sites defaults.from further testing I notice that zooming-in that webpage causes the issue to go away. Similiarly by changing the font-size on my desktop from 10 to 11 (not sure the units' used). counter-intuitively, other-sizes above 11 I tried replicated the issue.
